src/plugins/fs/mount.c

```diff
@@ -343,6 +343,9 @@ static gboolean get_mount_error_new (struct libmnt_context *cxt, int rc, MountAr
         if (permission)
             g_set_error (error, BD_FS_ERROR, BD_FS_ERROR_AUTH,
                          "Operation not permitted.");
+        else if (syscall_errno == ENODEV)
```

Comment:
You should add this to `get_mount_error_old` too. Nobody is going to use ntfs3 with the old libmount, but we should still make sure the errors are same for both cases.